security/remote_devices/spiffe_and_ssh/remote/remote-spire-agent/Dockerfile

```diff
@@ -17,35 +17,17 @@
 
 # Build utility container
 ARG BUILDER_BASE=golang:1.20-alpine3.17
```

Comment:
This can be removed also since it is no longer reference below.

Reply:
Including comment above this line.

Reply:
I was thinking to leave this in, because it might cause an error if CI tries to override it and there isn't an arg to override.